[2025-09-30T17:43:07.348+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2025-09-30T17:43:07.388+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: dag3.task_pwd scheduled__2025-09-30T17:42:00+00:00 [queued]>
[2025-09-30T17:43:07.399+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: dag3.task_pwd scheduled__2025-09-30T17:42:00+00:00 [queued]>
[2025-09-30T17:43:07.400+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 1
[2025-09-30T17:43:07.423+0000] {taskinstance.py:2330} INFO - Executing <Task(PythonOperator): task_pwd> on 2025-09-30 17:42:00+00:00
[2025-09-30T17:43:07.434+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61: DeprecationWarning: This process (pid=305) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2025-09-30T17:43:07.438+0000] {standard_task_runner.py:63} INFO - Started process 309 to run task
[2025-09-30T17:43:07.437+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'dag3', 'task_pwd', 'scheduled__2025-09-30T17:42:00+00:00', '--job-id', '67', '--raw', '--subdir', 'DAGS_FOLDER/dag3.py', '--cfg-path', '/tmp/tmptz3zwh6i']
[2025-09-30T17:43:07.445+0000] {standard_task_runner.py:91} INFO - Job 67: Subtask task_pwd
[2025-09-30T17:43:07.472+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/settings.py:195: DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
  SQL_ALCHEMY_CONN = conf.get("database", "SQL_ALCHEMY_CONN")

[2025-09-30T17:43:07.565+0000] {task_command.py:426} INFO - Running <TaskInstance: dag3.task_pwd scheduled__2025-09-30T17:42:00+00:00 [running]> on host 7cd44293b4c0
[2025-09-30T17:43:07.735+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='dag3' AIRFLOW_CTX_TASK_ID='task_pwd' AIRFLOW_CTX_EXECUTION_DATE='2025-09-30T17:42:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-09-30T17:42:00+00:00'
[2025-09-30T17:43:07.740+0000] {taskinstance.py:430} INFO - ::endgroup::
[2025-09-30T17:43:07.743+0000] {logging_mixin.py:188} INFO - 1
[2025-09-30T17:43:07.744+0000] {logging_mixin.py:188} INFO - 2
[2025-09-30T17:43:07.745+0000] {logging_mixin.py:188} INFO - 3
[2025-09-30T17:43:07.745+0000] {logging_mixin.py:188} INFO - 4
[2025-09-30T17:43:07.747+0000] {logging_mixin.py:188} INFO - 5
[2025-09-30T17:43:07.747+0000] {logging_mixin.py:188} INFO - 6
[2025-09-30T17:43:07.748+0000] {logging_mixin.py:188} INFO - 7
[2025-09-30T17:43:07.749+0000] {logging_mixin.py:188} INFO - 8
[2025-09-30T17:43:07.750+0000] {logging_mixin.py:188} INFO - 9
[2025-09-30T17:43:07.750+0000] {python.py:237} INFO - Done. Returned value was: None
[2025-09-30T17:43:07.754+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2025-09-30T17:43:07.781+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=dag3, task_id=task_pwd, run_id=scheduled__2025-09-30T17:42:00+00:00, execution_date=20250930T174200, start_date=20250930T174307, end_date=20250930T174307
[2025-09-30T17:43:07.855+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2025-09-30T17:43:07.897+0000] {taskinstance.py:3503} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2025-09-30T17:43:07.900+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
